### PR TITLE
OIDC-SSO

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: articles/sso/oidc/single-page-apps-sso.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+To bypass displaying the Lock screen when logging in a user (a process known as [*silent authentication*](#silent-authentication)), you must:
+
+* Enable the **Use Auth0 instead of the IdP to do Single Sign On** flag in the [Auth0 Client's settings page](${manage_url}/#/clients/${account.clientId}/settings);
+* Have a SSO cookie for the tenant's domain (in other words, the user has previously signed in and their saved cookie is still valid);
+* Pass the name of the user's Connection to Auth0 for authentication. You can do this by:
+  * Including it as a parameter when calling the `signin` function of the [auth0.js library](/libraries/auth0js);
+  * Passing the `connection` query string parameter when calling the [Authentication API's `/authorize` endpoint](/api/authentication#implicit-grant).
+
+### Initiate a Silent Authentication Request
```
Comment (Annyv2) [inline articles/sso/oidc/single-page-apps-sso.md]: Can a comment about [renewAuth](https://github.com/auth0/auth0.js#api) be added for people using Auth0.js?

Comment (khorne3) [reply articles/sso/oidc/single-page-apps-sso.md]: @Annyv2 Done. Thanks.

**Comment by mpaktiti** [comment]
Hey @khorne3 , can you please do the following:
- rebase to resolve conflicts
- name versions `legacy` and `current` (like this one https://github.com/auth0/docs/pull/4293 )
- make `current` the default version

Thanks!

**Comment by khorne3** [comment]
@mpaktiti All done. Thanks!

**Pull request merged**